src/pdf.js

```diff
@@ -164,4 +165,6 @@ export {
   renderTextLayer,
   // From "./display/svg.js":
   SVGGraphics,
+  // From "./scripting_api/scripting.js"
```

Comment:
`code_gen.js`

Moreover, I think it's more clear to call the file `code_generator.js` (we usually avoid abbrevations for readability).